src/app/_app.theme.scss

```diff
@@ -0,0 +1,12 @@
+@mixin app-theme($theme) {
+  // Extract the palettes you need from the theme definition.
+  $primary: map-get($theme, primary);
+  $accent: map-get($theme, accent);
+
+  // Define any styles affected by the theme.
+  .app-header {
```

Comment:
I just picked this to test/show how this works. Let me know if you have a better idea on where we could have an app-specific style